lib/api/multiObjectDelete.js

```diff
@@ -255,6 +256,9 @@ function getObjMetadataAndDelete(authInfo, canonicalID, request,
         ], (err, objMD, deleteInfo, versionId) => {
             if (err === skipError) {
```

Comment:
A minor style comment: the code flow could be slightly improved with the following structure to avoid repeating `return moveOn()`:

```
if (err) {
    if (err === skipError) {
    } else if (...) {
    } else {
        //generic error
    }
    return moveOn();
}
```